index.js

```diff
@@ -14,4 +14,150 @@ menuItem.forEach((menuItem) => {
+    image: './img/Dbg3.png',
+    stack: ['FACEBOOK', 'Full Stack Dev', '2015'],
+    technologies: ['HTML', 'CSS', 'JavaScript', 'Ruby', 'Bootstrap'],
+    liveLink: '',
+    sourceLink: '',
+  },
+  {
+    name: 'UBER Navigation',
+    description: 'Lorem Ipsum is simply dummy text of the prinhe 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.',
+    image: './img/Dbg4.png',
+    stack: ['FACEBOOK', 'Full Stack Dev', '2015'],
+    technologies: ['HTML', 'CSS', 'JavaScript', 'Ruby', 'Bootstrap'],
+    liveLink: '',
+    sourceLink: '',
+  },
+];
+
+const cardsContainer = document.querySelector('#About');
+
+projects.forEach((project) => {
+  const card = document.createElement('div');
+  card.classList.add('about-section');
+
+  card.innerHTML = `
+    <img src="${project.image}" class="card-img" />
+      <div class="canopy">
+        <h4 class="about-head">${project.name}</h4>
+        <ul class="cano">
+          <li class="tec">FACEBOOK</li>
+          <li class="tech">Full Stack Dev</li>
+          <li class="tech">2015</li>
+        </ul>
+        <p class="canopy-txt">
+        A daily selection of privately personalized reads; no accounts or
+          sign-ups required
+        </p>
+        <ul class="tech1">
+          <li class="tech-t">HTML</li>
+          <li class="tech-r">Ruby On Rails</li>
+          <li class="tech-t">CSS</li>
+          <li class="tech-j">JavaScript</li>
+        </ul>
+        
+        <button type="button" class="project-btn">See Project</button>
+      </div>
+  `;
+
+  cardsContainer.appendChild(card);
+
+  const button = card.querySelector('.project-btn');
+
+  const modalContainer = document.createElement('div');
+  modalContainer.classList.add('modal');
+  modalContainer.id = 'modal';
+
+  modalContainer.innerHTML = `
+  <div class="modal-content">
+    <span class="close">&times;</span>
+    <h2 class="project-title" id="projectTitle"></h2>
+    <ul id="cano">
+    </ul>
+    <div class="project-details">
+      <img class="project-image" id="projectImage" src="" alt="">
+      <div class="p-cont">
+      <p class="project-description" id="projectDescription"></p>
+      <div class="p-div">
+      <ul class="project-technologies" id="projectTechnologies"></ul>
+      <div class="project-buttons">
+        <button class="btn-link"><a class="project-live-link" id="projectLiveLink" href="#" target="_blank">See Live <img src="img/Icon.png" class="btn-img"/></a></button>
+        <button class="btn-link"><a class="project-source-link" id="projectSourceLink" href="#" target="_blank">See Source <img src="img/Frame.png" class="btn-img"/></a></button>
```

Comment:
Well! Your button looks good. 👍 but as per the design, the code needs some adjustment. Please consider updating the code to update the button's style to match the design specifications.

Additionally, it is not necessary to wrap the "a" tag with the "button" tag. You should choose either the "a" tag or the button tag, but using both is not necessary. Please update the code accordingly to ensure consistency and simplify the markup.

By making these adjustments, you will ensure better adherence to the design and improve the overall user experience.💪🔧🎨